extensions/flyway/runtime/src/main/java/io/quarkus/flyway/runtime/FlywayRecorder.java

```diff
@@ -69,6 +70,10 @@ public Flyway get() {
 
     public void doStartActions() {
         for (FlywayContainer flywayContainer : FLYWAY_CONTAINERS) {
+            if (flywayContainer.isBaselineAtStart() &&
+                    Objects.isNull(flywayContainer.getFlyway().info().current())) {
+                flywayContainer.getFlyway().baseline();
+            }
```

Comment:
I think we should analyze the order on how this change affects the others `*AtStart` flags. 
For example: 
- Do we have any mutual exclusion? 
- Should `clean()` be always executed first?